pages.it/common/git-add.md

```diff
@@ -0,0 +1,24 @@
+# git add
+
+> Aggiungi file nuovi o modificati all'indice.
```

Comment:
would it be more clear referring to the "area di stage" instead of the "indice" ?
Something like: "Aggiungi file nuovi o modificati all'area di stage." , "Aggiungi un file all'area di stage".
I'm probably going to refer to the `stage` area again for other commands

Reply:
Well, from [the italian doc](https://git-scm.com/book/it/v1/Per-Iniziare-Basi-di-Git):

> L'area di stage è un file, contenuto generalmente nella directory di Git, con tutte le informazioni riguardanti la tua prossima commit. **A volte viene indicato anche come 'indice', ma lo standard è definirlo come 'area di stage' (area di sosta, ndt)**.

So yea, go for it!